### DISCUSSION TITLE
How to enable debug logging with the on-prem docker container?

### DISCUSSION THREAD

**Discussion by deepankarm** [deepgram/community#803] [General help] [answered]
I'd like to look at debug logs to do some debugging with streaming audio on-prem. How to add debug logs with `quay.io/deepgram/onprem-engine` and `quay.io/deepgram/onprem-api` docker images?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
Hi @deepankarm 

It looks like you have a premium plan, so I encourage you to reach out to your support team. Customers with a Premium or VIP Support Plan with Deepgram can find details and links to contact us for support on your [Console dashboard](https://console.deepgram.com/).
